### PR TITLE
Intersil's DF Driver Wrapper

### PR DESCRIPTION
**Pull request by bugobliterator**
TODO: add commit to bump to DF master with https://github.com/PX4/DriverFramework/pull/61 pulled in.

### CODE REVIEW

**Comment by julianoes** [comment]
Please run:
```
./Tools/fix_code_style.sh src/platforms/posix/drivers/df_isl_wrapper/df_isl_wrapper.cpp
```

File: posix-configs/eagle/flight/px4.config [hunk 1]
```diff
@@ -8,6 +8,7 @@ df_mpu9250_wrapper start
 df_bmp280_wrapper start
 df_hmc5883_wrapper start
 df_trone_wrapper start
+#df_isl_wrapper start
```
Comment (julianoes) [inline posix-configs/eagle/flight/px4.config]: Could we enable it by default and it just exits if it doesn't find something with a timeout?

File: src/platforms/posix/drivers/df_isl_wrapper/df_isl_wrapper.cpp [hunk 2]
```diff
@@ -0,0 +1,394 @@
...
+int
+df_isl_wrapper_main(int argc, char *argv[])
+{
+	int ch;
+	int ret = 0;
+	int myoptind = 1;
+	const char *myoptarg = NULL;
+
+	/* jump over start/off/etc and look at options first */
+	while ((ch = px4_getopt(argc, argv, "R:", &myoptind, &myoptarg)) != EOF) {
```
Comment (julianoes) [inline src/platforms/posix/drivers/df_isl_wrapper/df_isl_wrapper.cpp]: There is no -R option right? So this can get removed?

**Comment by julianoes** [comment]
2 comments, other than that it looks fine.

File: posix-configs/eagle/flight/px4.config (hunk 1 above)
Comment (bugobliterator) [inline posix-configs/eagle/flight/px4.config]: @julianoes we don't more than one distance sensors, so enabling both can create some nastiness. I think selections made before runtime tend to be more robust.

File: src/platforms/posix/drivers/df_isl_wrapper/df_isl_wrapper.cpp (hunk 2 above)
Comment (bugobliterator) [inline src/platforms/posix/drivers/df_isl_wrapper/df_isl_wrapper.cpp]: @julianoes will do. thanks!

**Comment by bugobliterator** [comment]
@julianoes I ran fix condestyle on df_isl_wrapper, not much changes there, also fixed the unused option issue.

File: posix-configs/eagle/flight/px4.config (hunk 1 above)
Comment (julianoes) [inline posix-configs/eagle/flight/px4.config]: But I would argue that a user doesn't have two connected at the same time anyway.

**Comment by julianoes** [comment]
Thanks @bugobliterator for the fixes, unfortunately after the renaming, the build was not completely happy:

```
CMake Warning (dev) at cmake/common/px4_base.cmake:313 (add_dependencies):
  Policy CMP0046 is not set: Error on non-existent dependency in
  add_dependencies.  Run "cmake --help-policy CMP0046" for policy details.
  Use the cmake_policy command to set the policy and suppress this warning.

  The dependency target "df_isl" of target
  "platforms__posix__drivers__df_isl_wrapper" does not exist.
Call Stack (most recent call first):
  src/platforms/posix/drivers/df_isl_wrapper/CMakeLists.txt:36 (px4_add_module)
This warning is for project developers.  Use -Wno-dev to suppress it.
```

I renamed everything all the way, and pushed it to: #4392.

**Pull request closed without merging**